### PR TITLE
SMS provider cleanup

### LINKED ISSUE #7: Clean up after new SMS provider change
- [x] Create env var for origin email address for default service provider and modify migrations accordingly
- [x] Create env var for origin SMS phone number for default service provider and modify migrations accordingly
- [x] Remove all references firetext and mmg SMS providers and clients
- [x] Ascertain whether need to keep/remove/modify the `supports_international` flag for notifications providers

### PR DESCRIPTION
**Pull request by rahearn**
Removes most references to MMG and all references to Firetext.

MMG examples are kept around in two situations where SNS implementation is not complete:

1. delivery receipts (work to be tracked/created under #152)
3. inbound SMS (work to be tracked under #156)

closes #7

### CODE REVIEW

**Review by stvnrlly — CHANGES_REQUESTED** [review]
Thanks for tackling this, and it's nice to see how this sets up the next steps for delivery receipts and inbound messages.

Just a couple questions about small details.

File: app/notifications/receive_notifications.py [hunk 1]
```diff
@@ -246,7 +136,9 @@ def has_inbound_sms_permissions(permissions):
     return set([INBOUND_SMS_TYPE, SMS_TYPE]).issubset(set(str_permissions))
 
 
-def strip_leading_forty_four(number):
-    if number.startswith('44'):
-        return number.replace('44', '0', 1)
+def strip_leading_plus_one(number):
```
[the remark this answers is not in the document]
Comment (rahearn) [reply app/notifications/receive_notifications.py:139]: I'm going to take a closer look at this method while working on #135

File: migrations/versions/0382_remove_old_sms_providers_.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+down_revision = '0381_encrypted_column_types'
+
+
+def upgrade():
+    op.execute("DELETE FROM provider_details WHERE identifier IN ('mmg', 'firetext')")
+    op.execute("DELETE FROM provider_details_history WHERE identifier IN ('mmg', 'firetext')")
+
+
+def downgrade():
+    raise Exception("Irreversible migration")
```
[the remark this answers is not in the document]
Comment (rahearn) [reply migrations/versions/0382_remove_old_sms_providers_.py:22]: We could put entries back into `provider_details` but entires in `provider_details_history` will be lost, which is why I went with irreversible.

In my experience, rollbacks don't generally happen that often, and even less likely to be doing a multi-revision downgrade, at least as long as you're deploying code anywhere close to regularly.

File: app/notifications/receive_notifications.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (stvnrlly) [reply app/notifications/receive_notifications.py:139]: It looks like my approach wouldn't work: those numbers are also 11 digits. I'll make a note over in that issue.

File: migrations/versions/0382_remove_old_sms_providers_.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (stvnrlly) [reply migrations/versions/0382_remove_old_sms_providers_.py:22]: My initial thought was that `provider_details_history` would be ok to lose, but you're right that this should be considered irreversible in any production system.

**Review by stvnrlly — APPROVED** [review]

**Pull request merged**